pkg/cmd/login/browser.go

```diff
@@ -30,41 +33,36 @@ type Handler struct {
 	state    string
 	baseURL  string
 	port     int
-	response chan string
+	response chan *types.User
 	errChan  chan error
 }
 
-// OriginData
-type OriginData struct {
-	Meta    string
-	Origin  string
-	Message string
-}
-
 func (h *Handler) handle() http.Handler {
 	fn := func(w http.ResponseWriter, r *http.Request) {
 		code := r.URL.Query().Get("code")
 		s := r.URL.Query().Get("state")
 
 		if h.state != s {
 			h.errChan <- fmt.Errorf("invalid request state")
-			w.WriteHeader(http.StatusBadRequest)
 			return
 		}
 
-		data := &OriginData{
-			Meta:    "http-equiv=\"Content-type\" content=\"text/html; charset=utf-8\"",
-			Origin:  "the Okteto CLI",
-			Message: "Close this window and go back to your terminal",
+		ctx := context.Background()
```

Comment:
```suggestion
		ctx := r.Context()
```

Reply:
```suggestion
		ctx := h.Context()
```

??

Reply:
is the one from the handler or the requesT?